### PR TITLE
feat: fix testrail update script bug

### PR DESCRIPTION
**Pull request by refael-m**
fix testrail update script bug throw exception when there is no new files

### CODE REVIEW

File: src/commons/GitUtils.ts [hunk 1]
```diff
@@ -12,7 +12,12 @@ export namespace GitUtils {
   export function getMergedTestsIdsSinceDay(sinceDay: number): Set<string> {
     console.log(`Getting tests id's since ${sinceDay} days ago`);
     const params: string = `diff $(git rev-list -n1  --before={@{${sinceDay}.days.ago} origin/master) --name-only`;
-    const filesSinceDay = executeGitScript(params).split(/[\r\n]+/);
+    const scriptRes = executeGitScript(params);
+    if (!scriptRes) {
+      console.log(`No changed files found since ${sinceDay} days ago`);
+      return undefined;
```
Comment (FelixZilber) [inline src/commons/GitUtils.ts:18]: better to return null in this case

File: src/scripts/updateTestRailTestsFromLastDay.js [hunk 2]
```diff
@@ -14,9 +14,13 @@ const DAYS_TO_COUNT_BACK = Number(process.env.DAYS_TO_COUNT_BACK) || 1;
  */
 (() => {
   const testsToUpdate = GitUtils.getMergedTestsIdsSinceDay(DAYS_TO_COUNT_BACK);
-  console.log(`Updating ${testsToUpdate.size} tests...`);
-  console.log(testsToUpdate);
+  if (!testsToUpdate || testsToUpdate.size === 0) {
```
Comment (FelixZilber) [inline src/scripts/updateTestRailTestsFromLastDay.js:17]: in what scenario the array size can be 0?

**Review by FelixZilber — APPROVED** [review]

File: src/scripts/updateTestRailTestsFromLastDay.js (hunk 2 above)
Comment (refael-m) [reply src/scripts/updateTestRailTestsFromLastDay.js:17]: When files that are not tests files merged last day

File: src/commons/GitUtils.ts (hunk 1 above)
Comment (refael-m) [reply src/commons/GitUtils.ts]: done

Comment (refael-m) [reply src/commons/GitUtils.ts]: done

**Pull request merged**